sorc/chgres_cube.fd/wam_climo_data.f90

```diff
@@ -0,0 +1,2758 @@
+!! @param [in] glong - geodetic longitude(deg)                                
+!! @param [in] stl - local apparent solar time(hrs; see note below)           
+!! @param [in] f107a - 81 day average of f10.7 flux (centered on day ddd)     
+!! @param [in] f107 - daily f10.7 flux for previous day                       
+!! @param [in] ap - magnetic index(daily) or when sw(9)=-1. :                 
+!!                - array containing:                                         
+!!                  (1) daily ap                                              
+!!                  (2) 3 hr ap index for current time                        
+!!                  (3) 3 hr ap index for 3 hrs before current time           
+!!                  (4) 3 hr ap index for 6 hrs before current time           
+!!                  (5) 3 hr ap index for 9 hrs before current time           
+!!                  (6) average of eight 3 hr ap indicies from 12 to 33 hrs pr
+!!                         to current time                                    
+!!                  (7) average of eight 3 hr ap indicies from 36 to 57 hrs pr
+!!                         to current time                                    
+!! @param [in] mass - mass number (only density for selected gas is           
+!!                calculated.  mass 0 is temperature.  mass 48 for all. 
+!!                mass 17 is anomalous o only.)                         
+!!                                                                      
+!!    Notes on input variables:                                         
+!!       ut, local time, and longitude are used independently in the    
+!!       model and are not of equal importance for every situation.     
+!!       for the most physically realistic calculation these three      
+!!       variables should be consistent (stl=sec/3600+glong/15).        
+!!       the equation of time departures from the above formula         
+!!       for apparent local time can be included if available but       
+!!       are of minor importance.                                       
+!!                                                                      
+!!       f107 and f107a values used to generate the model correspond    
+!!       to the 10.7 cm radio flux at the actual distance of the earth  
+!!       from the sun rather than the radio flux at 1 au. the following 
+!!       site provides both classes of values:                          
+!!       ftp://ftp.ngdc.noaa.gov/stp/solar_data/solar_radio/flux/       
+!!                                                                      
+!!       f107, f107a, and ap effects are neither large nor well         
+!!       established below 80 km and these parameters should be set to  
+!!       150., 150., and 4. respectively.                               
+!!                                                                      
+!! output variables:                                                 
```

Comment:
Please put all param tags at the bottom of the docblock, just above the author, like this:

```
!> Short sentence to describe the subprogram.
!!
!! Many paragraphs of description can be included. You can go
!! on and on.
!! 
!! Use an empty documentation line as a paragraph break.
!!
!! @param[in] a description of a
!! @param[out] b decription of b
!! @author Freddy Fortran
subroutine fun1(a, b)

```
All parameters need to be in the order they appear in the subprogram argument list. The parameters should not be ordered differently in the documentation and the code, that is confusing as heck to the reader.

No need for lines like:
`output variables:
`
Doxygen knows how to format the params.

Reply:
OK, deleted.